src/legacy/core_plugins/data/public/search/aggs/agg_configs.ts

```diff
@@ -90,37 +85,8 @@ export class AggConfigs {
 
     this.aggs = [];
     this.indexPattern = indexPattern;
-    this.schemas = opts.schemas;
 
     configStates.forEach((params: any) => this.createAggConfig(params));
-
-    if (this.schemas) {
-      this.initializeDefaultsFromSchemas(this.schemas);
```

Comment:
the defaults are no longer initialized on this level, this is moved into vis_impl.js as its visualization specific